pkg/azureconstants/azure_constants.go

```diff
@@ -122,6 +122,7 @@ const (
 	VolumeAttachRequestAnnotation         = "disk.csi.azure.com/volume-attach-request"
 	VolumeDeleteRequestAnnotation         = "disk.csi.azure.com/volume-delete-request"
 	VolumeDetachRequestAnnotation         = "disk.csi.azure.com/volume-detach-request"
+	CloudDeleteVolume                     = "cloud-delete-volume"
 	RecoverAnnotation                     = "disk.csi.azure.com/recovery" // used to ensure reconciliation is triggered for recovering CRIs
 	VolumeNameLabel                       = "disk.csi.azure.com/volume-name"
 	VolumeIDLabel                         = "disk.csi.azure.com/volume-id"
```

Comment:
It looks like we no longer use `PreProvisionedVolumeCleanupAnnotation`. It should be removed.